aiida_vasp/workchains/relax.py

```diff
@@ -313,6 +313,9 @@ def verify_next_workchain(self):
             self.ctx.exit_code = compose_exit_code(next_workchain_exit_status, next_workchain_exit_message)
             self.report('The called {}<{}> returned a non-zero exit status. '
                         'The exit status {} is inherited'.format(workchain.__class__.__name__, workchain.pk, self.ctx.exit_code))
+            # Make sure at the very minimum we attach the misc node that contains notifications and other
+            # quantities that can be salvaged
+            self.out('misc', workchain.outputs['misc'])
```

Comment:
This is a good point. Maybe we should just attach all outputs of the downstream workchain and salvage as much as we can?

Reply:
Yes, I do not know yet. Possibly. I think I will also remove it and only pull it up to the `VaspWorkChain` for now. Already now there is garbage in `misc`. Suspect if we salvage all output it would yield a lot of nodes with `null/None` etc. which could clutter up the analysis and query later if you do not account for this. I suggest we leave it as is for now and then we can revisit this in the future if we also need to salvage everything else. I thought for a moment that it would be better to have separate nodes for this particular reason, e.g. that we only salvage the `notifications`. But we might also put some statistics in `misc` which we might want to use for error correction etc. as well, so it seems to make sense to have all this in one container. Okey?